modules/nginxvts/charts.go

```diff
@@ -0,0 +1,118 @@
+package nginxvts
+
+import "github.com/netdata/go.d.plugin/agent/module"
+
+type (
+	// Charts is an alias for module.Charts
+	Charts = module.Charts
+	// Dims is an alias for module.Dims
+	Dims = module.Dims
+)
+
+var nginxVtsMainCharts = Charts{
+	{
+		ID:    "times",
+		Title: "Nginx running time( Uptime=(nowMsec-loadMsec)/1000 )",
+		Units: "milliseconds",
+		Fam:   "main",
+		Ctx:   "nginxvts.times",
+		Dims: Dims{
+			{ID: "loadmsec", Name: "load"},
+			{ID: "nowmsec", Name: "up"},
+		},
+	},
+	{
+		ID:    "connections",
+		Title: "Nginx Connections",
+		Units: "connections",
+		Fam:   "main",
+		Ctx:   "nginxvts.connections",
+		Dims: Dims{
+			{ID: "connections_active", Name: "active"},
+			{ID: "connections_reading", Name: "reading"},
+			{ID: "connections_writing", Name: "writing"},
+			{ID: "connections_waiting", Name: "waiting"},
+			{ID: "connections_accepted", Name: "accepted"},
+			{ID: "connections_handled", Name: "handled"},
+			{ID: "connections_total", Name: "total"},
```

Comment:
The "total" is from "connections.requests". It can be calculated by adding by all "requestCounter" in serverZones. 
According to my test，active+reading+...+ handled does not equal to requests. How about split this chart into 3 charts: 

- active connections (active)
- connections by state (all other - dimensions algorithm - Incremental - we want to show rate, not counter, units - connections/s).
- total requests

Reply:
- `connections.requests`: The total number of requested client connections.
- `serverZones[*].requestCounter`: The total number of client requests received from clients.

Same counter?

(removed not related data)

```json
"connections": {
        "requests": 7203
    },
    "serverZones": {
        "*": {
            "requestCounter": 7202,
            "responses": {
                "2xx": 7202,
```